jupyterlab_ucaip/src/components/base_widget.tsx

```diff
@@ -0,0 +1,34 @@
+import { ReactWidget } from '@jupyterlab/apputils';
+import { createMuiTheme, ThemeProvider } from '@material-ui/core/styles';
+import { COLORS } from 'gcp_jupyterlab_shared';
+import * as React from 'react';
+import { AppContext, Context } from '../context';
+
+const theme = createMuiTheme({
+  palette: {
+    primary: {
+      main: COLORS.blue,
+    },
+    secondary: {
+      main: '#747474',
+    },
+  },
+});
+
+export abstract class BaseWidget extends ReactWidget {
+  constructor(protected context: Context) {
+    super();
+  }
+
+  render() {
+    return (
+      <ThemeProvider theme={theme}>
+        <AppContext.Provider value={this.context}>
+          {this.body()}
+        </AppContext.Provider>
+      </ThemeProvider>
```

Comment:
This is also a really good idea in terms of changing the mui theme https://material-ui.com/customization/palette/#dark-mode